CHANGELOG.md

```diff
@@ -6,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased][]
 
+## [1.0.5][] - 2018-11-19
```

Comment:
Don't we have a script in place that updates CHANGELOG.md and package.json when a new release is made @tomdaniels?

Reply:
@jonathanweiss yes we have this script in place - might be worthwhile eventually moving to semantic-release for OSS though (but thats another story :) )